docs/help-topics/appstudio-incontext-help/appstudio-incontext.yml

```diff
@@ -0,0 +1,167 @@
+# Names are UI elements names.
+# Tags to be kept empty for now. Tags will specify where in the app descriptions will be available.
+# Titles are "dictionary" articles titles.
+# Links to be external only. We don't know yet whether referencing to other terms within this list will be supported.
+
+# Terms that are 100% familiar to developers and probably shouldn't be part of this list: API token, Target port, Build command, Run command
+
+# DTUX-1136 Final design https://docs.google.com/document/d/1dMO7oCLdgayoNa3HvpNhvsGhBk9MoGG5TUnvbq8wDjs
+
+- name: Git repo URL or container image
+  tags:
+  title: Git repo URL or container image
+  content: |-
+    You can build your application using the source code from your Git repository or a container image.
+    * To import code from a Git hosting service, provide the URL to your private or public Git repository. If App Studio can't reach the repository, authorize access to it.
+    * To import code from a container image, provide a link to a private or public container registry. If App Studio can't reach the registry, authorize access to it.
+  links:
+    "[Link title]()"
+
+- name: Start with a sample
+  tags:
+  title: Sample application
+  content: |-
+    App Studio provides a catalogue of basic sample applications. You can't modify the settings of sample applications.
+  links:
+    "[Link title]()"
+
+# Authorization section, option 1, Authorization and Git options
+- name: Authorize Git repository
+  tags:
+  title: Git repository access authorization
+  content: |-
+    You can sign in to your private Git repository, a login window will open in a separate tab. Alternatively, you can use API token authentication. Maybe a list of supported Git hosting services?
+  links:
+    "[Link title]()"
+
+# Authorization section, option 2, Authorization
+- name: Authorize container registry
+  tags:
+  title: : Container registry authorization
```

Comment:
You have an extra `:`. That will break the DB seeding.

Reply:
Good catch, thank you!